docusaurus.config.ts

```diff
@@ -453,6 +453,10 @@ const config: Config = {
           }
         : undefined,
     },
+    tableOfContents: {
+      minHeadingLevel: 2,
+      maxHeadingLevel: 4,
+    },
```

Comment:
Any particular reason for changing this?

If you only want to change it for a single article, you can add `toc_min_heading_level` and `toc_max_heading_level` to the frontmatter instead [(see docs)](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#toc_min_heading_level).

Reply:
It's a nice way to structure the right nav-bar, since it will list an extra level of depth. I can limit it to this file if you prefer, but I see it as a very useful change for all docs. It just helps to navigate to the section you need instead of scrolling in the doc, and it also gives more clarity of the content of the doc since more things are listed.